user-docs/modules/ROOT/pages/fedora-iot-bootc-quay-example.adoc

```diff
@@ -0,0 +1,128 @@
+= Fedora IoT Bootc Image Example with Quay
+
+== Building and Booting a Fedora IoT Bootc Image
+
+This example walks through building and booting a Fedora IoT bootc image using Quay.io, as well as pushing an update to a booted 
+Fedora IoT system.
+
+To start, clone the https://gitlab.com/fedora/bootc/base-images[Fedora Bootc Base Images] Git repository. This includes the Fedora
+IoT package manifest we will use to create our Fedora IoT bootc image.
+
+Within this clone, create a `Containerfile.custom` with the following contents:
+----
+FROM localhost/fedora-bootc as builder
+RUN /usr/libexec/bootc-base-imagectl build-rootfs --manifest=fedora-iot /target-rootfs
+
+FROM scratch
+COPY --from=builder /target-rootfs/ /
+LABEL containers.bootc 1
+ENV container=oci
+STOPSIGNAL SIGRTMIN+3
+CMD ["/sbin/init"]
+----
+
+Next, build a `localhost/fedora-bootc` image using the command below. This is required for building your Fedora IoT bootc image, as the 
+containerfile you created uses `localhost/fedora-bootc` as builder:
+----
+sudo podman build --cap-add=all --security-opt=label=disable \
+--device /dev/fuse -t localhost/fedora-bootc .
+----
+
+Now tag that `localhost/fedora-bootc` image so you can push it to Quay.io:
+----
+sudo podman tag localhost/fedora-bootc:latest quay.io/[repository-name]:fedora-bootc
+----
+
+Before pushing to Quay.io, you may need to log in:
+----
+sudo podman login quay.io
+----
+
+Push your image to your Quay.io repository:
+----
+sudo podman push quay.io/[repository name]:fedora-bootc
+----
+
+You're now ready to build a Fedora IoT bootc image using the custom containerfile you made earlier. Use this command:
+----
+sudo podman build --cap-add=all --security-opt=label=disable \
+--device /dev/fuse -t localhost/fedora-iot -f Containerfile.custom 
+----
+
+Then, tag your Fedora IoT bootc image:
+----
+sudo podman tag localhost/fedora-iot:latest quay.io/[repository name]:fedora-iot
+----
+
+Push your new Fedora IoT bootc image to Quay.io using the following command. Note that you may need to log in again:
+----
+sudo podman push quay.io/[repository name]:fedora-iot
+----
+
+Now, boot your Fedora IoT bootc image. Open a new terminal window and run:
+----
+podman-bootc run --filesystem=ext4 quay.io/[repository name]:fedora-iot
```

Comment:
I don't believe that `podman-bootc` is packaged in Fedora...

You may want to link to the upstream repo where the install instructions are or change this section to cover building a disk image with `bootc-image-builder` and provisioning a VM